src/computer_session.h

```diff
@@ -17,10 +20,13 @@ class computer_session
 
     private:
         computer &comp;
-        // Output window
-        catacurses::window w_terminal;
-        // Pretty border
-        catacurses::window w_border;
+        // Output window. This class assumes w's dimensions do not change.
+        const catacurses::window w;
```

Comment:
Maybe you should replace this variable name with a more understandable name? For example `terminal_window` or `computer_window`.

Reply:
I think `win` will be clear enough.